gen-resourcesdocs/pkg/config/output.go

```diff
@@ -253,16 +244,16 @@ func (o *TOC) OutputOperations(i int, outputChapter outputs.Chapter, operations
 		return err
 	}
 	for i, operation := range *operations {
-		o.OutputOperation(i, operationsSection, &operation)
-		_ = operation
```

Comment:
I presume this `_ = operation` line was to keep Go from complaining about an unused variable, which is not relevant here (anymore), since `operation` is being used.